Lib/test/test_docxmlrpc.py

```diff
@@ -188,9 +188,9 @@ def test_annotations(self):
                      b'<dd><tt>Use&nbsp;function&nbsp;annotations.</tt></dd>')
         self.assertIn(
             (b'<dl><dt><a name="-annotation"><strong>annotation</strong></a>'
-             b'(x: int)</dt>' + docstring + b'</dl>\n'
+             b'(x: \'int\')</dt>' + docstring + b'</dl>\n'
              b'<dl><dt><a name="-method_annotation"><strong>'
-             b'method_annotation</strong></a>(x: bytes)</dt></dl>'),
+             b'method_annotation</strong></a>(x: \'bytes\')</dt></dl>'),
```

Comment:
I don't think this is "as expected". It seems this is due to the change in behavior for `inspect.signature`. That worries me.